docs/01. installation/07. docker.md

```diff
@@ -124,61 +146,93 @@ You can easily use different credentials by adjusting them in the `docker-compos
 - Finish the installation process. That's it, enjoy your fresh local Fork CMS installation powered by Docker.
 
 
-## Configuring your local develop environment in PHPStorm
-### XDebug
-XDebug is one of the most popular tools to debug your PHP application. It's easy to set up in PHPStorm. 
+## PHPStorm configuration
+
+### General setup and XDebug
+[Video - watch how to setup PHPStorm and Xdebug](https://d.pr/v/Xq1uYJ)
+
+It's easy to setup PHPStorm in combination with Docker and Xdebug, one of the most popular tools to debug your PHP application.
 
-- Copy the .env.dist to .env
+- First copy the .env.dist to .env in the root folder of your project
 ```
 cp .env.dist .env
 ```
 
-- You can edit the `.env` file to fill in some host-specific information and configure XDebug. 
-- Restart/rebuild your Docker containers, because you modified the environment variables:
+- You can edit the `.env` file to fill in some host-specific information and configure XDebug. By default it contains a special
+hostname available for Docker for Mac users, but you can change it to point to your own Docker host IP.
+- Recreate your Docker containers, because you modified the environment variables.
 ```
-docker-compose build
+docker-compose down
 docker-compose up -d
 ```
 
-- In PHPStorm's preferences, go to the PHP > Debug section. Make sure Xdebug accepts connections and listens on port 9000 (should be default configuration).
-![Configuring XDebug in PHPStorm](assets/phpstorm-xdebug1.png)
+- In PHPStorm's preferences, go to "Build, Execution, Deployment > Docker". Make sure you create a Docker configuration that 
+connects to the Docker daemon "Docker for mac". 
+
+- We also need to configure PHPStorm to use the PHP interpreter that resides inside the app container. To do so, go to 
+preferences to "Languages & Frameworks > PHP". Click the "..." icon for the PHP interpreter. Create a new interpreter
+and use type "Docker". Select the Docker for mac server you created, and the Docker image name for the project you're working on.
+Make sure PHPStorm detects your php version correctly (see video above). Next, configure the Docker container by clicking the "...",
+and fill in the Network mode with the value you find when running `docker network ls`. Next, go to Volume bindings and
+make sure the container path is set to `/var/www/html`. This is the path where our code lives inside the container. 
+
+- Let's configure Xdebug now. In PHPStorm's preferences, go to the PHP > Debug section. Make sure Xdebug accepts connections and 
+listens on port 9000 (should be default configuration). ![Configuring XDebug in PHPStorm](assets/phpstorm-xdebug1.png)
```

Comment:
@forkcms/core-contributors can one of the PHPStorm users follow these instructions and confirm that it's working and easy to follow?

Seems clear to me at first sight though.

Reply:
👍  easiest would be to follow along with the video. Let me know any remarks while setting it up